### ISSUE TITLE
can you please push 4.1.0 to npm

### ISSUE THREAD

**Issue by sarkistlt** [feathers-plus/feathers-hooks-common#326]
current latest version is 3.10.0, it doesn't have some of the hooks such a `cache`

**Comment by bsubedi26** [comment]
@sarkistlt, As noted in the docs  under Usage(https://feathers-plus.github.io/v1/feathers-hooks-common/#Usage), you can use it directly by installing the master branch repo.
``` 
Until its published on npm, you can get install v4.0.0 with "feathers-hooks-common": "git://github.com/feathers-plus/feathers-hooks-common.git#master" in package.json.
```

**Comment by sarkistlt** [comment]
I see, thanks for pointing to the link
